scripts/export_database.pl

```diff
@@ -265,6 +265,10 @@ sub sanitize_field_content {
 		$csv .= "${nid}_100g" . "\t";
 	}
 
+	if (grep { $_ eq 'products_obsolete' } @collections) {
+		$csv .= "obsolete\t";
+	}
```

Comment:
collections is defined below !

I think you can just add the column in every cases.

Reply:
If fact you don't need this if you correctly add obsolete to @export_fields (in Config_xxx.pm)